### PR TITLE
MediaPipe Tasks Python Image Classifier Benchmark

### PR DESCRIPTION
**Pull request by kinaryml**

### CODE REVIEW

File: mediapipe/tasks/python/benchmark/vision/image_classifier/image_classifier_benchmark.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import numpy as np
+import mediapipe as mp
+from mediapipe.tasks import python
+from mediapipe.tasks.python import vision
+
+_IMAGE_FILE = 'burger.jpg'
+
+
+def run(model: str, n_iterations: int, delegate: python.BaseOptions.Delegate):
+    """Run asynchronous inference on images and benchmark.
```
Comment (schmidt-sebastian) [inline mediapipe/tasks/python/benchmark/vision/image_classifier/image_classifier_benchmark.py:26]: Please document the return value (and maybe allow us to pass in the percentile to use?)

Comment (kinaryml) [reply mediapipe/tasks/python/benchmark/vision/image_classifier/image_classifier_benchmark.py:13]: I've fixed it now.

**Review by schmidt-sebastian — APPROVED** [review]

File: mediapipe/tasks/python/benchmark/vision/image_classifier/setup.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Install Python dependencies.
```
Comment (schmidt-sebastian) [inline mediapipe/tasks/python/benchmark/vision/image_classifier/setup.sh:1]: Note that I removed this file as part of the import. I changed your script to build via Bazel (added a BUILD file). I moved the commands to download the models to the README. Building via Bazel allows us to ensure that these script don't break as we change our CI.

Comment (kinaryml) [reply mediapipe/tasks/python/benchmark/vision/image_classifier/setup.sh:1]: Agreed and it ensures we're pulling the updated models while benchmarking.

**Pull request merged**